src/services/lnd/auth.ts

```diff
@@ -24,7 +26,7 @@ export const addProps = (array) =>
       // find a way to mock this up for jest
       // for now only trigger is active = false at start
       // because trigger will start listening to lnd event on lnd start/restart
-      active: !isTrigger,
+      active: isSpec || !isTrigger,
```

Comment:
(I know I already ask the question about the filename but didn't dive into it)

what does spec stands for? test?

Reply:
yes, test, do you prefer `isTest`?

Reply:
changed to isTest https://github.com/GaloyMoney/galoy/pull/428/commits/20668101e2a16ba4ad949eaa1af8980a7bbc26ed